frontend/src/concepts/pipelines/topology/pipelineTaskTypes.ts

```diff
@@ -31,7 +31,7 @@ export type PipelineTaskRunStatus = {
   startTime: string;
   completeTime?: string;
   podName?: string;
-  state?: RuntimeStateKF;
+  state?: string;
```

Comment:
I think we should try to keep the state an enum here unless there's a good reason not to.